nametoaddr.c

```diff
@@ -126,7 +126,14 @@ pcap_nametoaddr(const char *name)
 	bpf_u_int32 **p;
 	struct hostent *hp;
 
+#ifdef _WIN32
+#pragma warning (push)
+#pragma warning (disable: 4996) /* disable MSVC's gethostbyname() warning here */
+	if ((hp = gethostbyname(name)) != NULL) {
+#pragma warning (pop)
```

Comment:
> Ok, but how about:

I think many source files have included ``pcap-stdinc.h``. Disabling this warning in this manner will still cause all warnings of ``_WINSOCK_DEPRECATED_NO_WARNINGS`` type to be disabled in those source files when building libpcap. This is also not good.

> Since the pragma is effective for MSVC only. And since the warning is not really a warning from the compiler. But a warning caused by recent changes in the MS-SDK.

I'm using MSVC 2013 for building libpcap. MSVC 2013 is also the official compiler for official WinPcap, Wireshark and Nmap. So I think MSVC 2013 can be represented as MSVC here.

> Have you tried building with e.g. MinGW and checked the warnings from these unknown pragmas?

Np. I didn't try to build on ``MinGW``. I don't know how it works on ``MinGW``. We can change it to ``#if defined(_WIN32) && defined (_MSC_VER)``.